input/pagecontent/reference.md

```diff
@@ -690,21 +690,21 @@ For a path to an element of an ElementDefinition within a StructureDefinition, u
 
 **Note:** When using caret paths to access metadata of element definitions, there is a REQUIRED space before the `^` character.
 
-A special case of the ElementDefinition path is setting properties of the first element of the differential (i.e., StructureDefinition.differential.element[0]). This element always refers to the profile or standalone extension itself. Since this element does not correspond to a named element appearing in an instance, the dot or full stop (`.`) SHALL be used to represent it (The dot symbol is often used to represent "current context" in other languages). It is important to note that the "self" elements are not the elements of a StructureDefinition directly, but elements of the first ElementDefinition contained in the StructureDefinition. The syntax is:
+A special case of the ElementDefinition path is setting properties of the first element of the differential (i.e., `StructureDefinition.differential.element[0]`). This element always refers to the profile or standalone extension itself. Since this element does not correspond to a named element appearing in an instance, the dot or full stop (`.`) SHALL be used to represent it (The dot symbol is often used to represent "current context" in other languages). It is important to note that the "self" elements are not the elements of a StructureDefinition directly, but elements of the first ElementDefinition contained in the StructureDefinition. The syntax is:
```

Comment:
I thought that was weird too, but assumed it was probably correct and I just didn't know how to use full sentence parenthetical sentences correctly. But now that you've said something, I'm changing it!

Reply:
Because I had to double check, I looked up some rules around parentheses. I think because "The dot symbol is often used to represent "current context" in other languages" is a complete sentence we could treat it like its own sentence, but we'd want to separate it from the sentence that comes before it. So something like this:

> Since this element does not correspond to a named element appearing in an instance, the dot or full stop (`.`) SHALL be used to represent it. (The dot symbol is often used to represent "current context" in other languages.)

But I'm fine if we want to just treat it like a clause and leave it as is but with the lower case "The". 

> Since this element does not correspond to a named element appearing in an instance, the dot or full stop (`.`) SHALL be used to represent it (the dot symbol is often used to represent "current context" in other languages).

Reply:
Oh. I think I like the separate sentence entirely. Why didn't I think of that?

Reply:
Fixed in 94e69762c29aa5bd288bab9afe242a68c14ba5c3.

Reply:
Oh shoot. I did it wrong. :-/

Reply:
Fixed in ba168d2b234806b37a6558f47db1ff41aa5c3951.